TodoApi.Tests/TodoApiTests.cs

```diff
@@ -0,0 +1,117 @@
+namespace TodoApi.Tests;
+using Xunit;
+using Moq;
+using Microsoft.AspNetCore.Mvc;
+using TodoApiWithControllers.Models;
+using TodoApiWithControllers.Services;
+using TodoApiWithControllers.Controllers;
+
+public class UnitTest1
+{
+    [Fact]
+    public async void PostTodoItem_CreatesATodoItem()
+    {
+        var mockTodoItemDTO = new Mock<TodoItemDTO>();

```

Comment:
You can use [this](https://github.com/bchavez/Bogus) to generate fake data.

Reply:
Aye, I'll check that out. Thank you